roles/tendrl-api/defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+# values: packages | source
+install_from: packages
+# when specified, tendrl-api is installed with provided repo support
+enable_repo: None
+# url of the source repository and the branch to checkout
+# e.g. tendrl_api_repo_url: git://github.com/Tendrl/tendrl.git
+tendrl_api_repo_url: https://github.com/Tendrl/tendrl-api.git
+tendrl_api_repo_branch: master
+# target (the git repo would be checked out there)
+tendrl_api_local_path: /opt/tendrl/tendrl-api
+tendrl_api_bundle_install_path: vendor/bin
+# if there some areas which we don't want to install set
+# tendrl_api_bundle_without to true
+tendrl_api_bundle_without: false
+# if true a tendrl_api_bundle_without_groups has to be set too
```

Comment:
It would be a good idea to group variables there into 2 or 3 piles, so that I'm not distracted by plethora of source installation knobs when tweaking installation from packages.